Lib/bdb.py

```diff
@@ -293,6 +396,10 @@ def _prune_breaks(self, filename, lineno):
             del self.breaks[filename]
 
     def clear_break(self, filename, lineno):
+        """Delete breakpoints in filename:lineno.
+
+        If none were set, return an error message.
```

Comment:
I think if `none were set` should be more clearly. But not sure how to change it.